packages/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,39 @@
+const knex = require('../../config/db');
```

Comment:
I'd try to avoid adding the same change in multiple commits/branches.
Either wait for work to be approved and merged, or point your branch to the branch that adds this work instead.